rest/serializers.py

```diff
@@ -289,3 +290,19 @@ class LearningResourceExportTaskSerializer(Serializer):
     id = CharField()
     status = CharField()
     url = CharField()
+
+
+class RepositorySearchSerializer(Serializer):
+    """Serializer for search metadata."""
+    text = CharField()
```

Comment:
So it is useless for the interface, because for the panel you need the version with the XML and probably you need to make another request anyway.